common/on_action/birthday.txt

```diff
@@ -6,6 +6,10 @@ on_birthday = {
 		on_birthday_childhood
 		on_graceful_aging_birthday
 	}
+	# Warcraft 
+	events = {
+		WCNIC.1
+	}
```

Comment:
Wait a second. This isn't a random event! It happens for all rulers who met the conditions!
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113171832-eca5d380-9258-11eb-8854-d22c1ccabb4b.png)

</details>

Reply:
done in d9b40895